### PR TITLE
Add support for Made With CNC Handheld Console

### PR DESCRIPTION
**Pull request by liqdfire**
Add userspace component to add support for a Made With CNC Handheld Console.

### CODE REVIEW

File: features/handheld_console/overlay.inc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[HANDHELD_CONSOLE]
+BUTTON_0_SIGNAL=feed-hold-3
```
Comment (jallwine) [inline features/handheld_console/overlay.inc]: I would use feed-hold-5, cycle-start-5 and abort-5 to be on the safe side. I know for sure we're using feed-hold-3 for our safety switch enclosure and feed-hold-4 for our high speed spindle machines. Users with those features could override them, but using 5 will avoid those conflicts.

File: features/handheld_console/handheld_console.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python3
```
Comment (jallwine) [inline features/handheld_console/handheld_console.py:1]: This file needs to be made executable.

**Review by jallwine — CHANGES_REQUESTED** [review]

**Review by jallwine — COMMENTED** [review]
- The selectAxis function doesn't adequately deselect the B axis because range(1,5) iterates over 1, 2, 3 and 4.
- The rotary axis increments are very small. Perhaps add options for customizing the increment values?
- At start up, the handheld_console pins don't sync properly.

**Comment by jallwine** [comment]
I think 36 is too high for a default base rotary increment. I would choose either 3.6 if you think a quarter turn of the dial should result in a 90 degree movement at the largest increment or 1 if the largest increment should be tied to the numbers on the dial (so a 90 degree movement when starting at 0 and turning the dial to 90).

I'll accept this pull request with either of those changes.

If you want to go the next step and allow a user to customize those values on the fly with M codes, the selected-linear-jog-incr and selected-rotary-jog-incr pins should probably get sync'd on a jog event rather than only in selectJogIncrement.

**Comment by liqdfire** [comment]
Modified the base jog increments for rotary axes to 1 degree.

**Pull request merged**